src/Town/js/createTown.ts

```diff
@@ -92,22 +92,10 @@ export const createTown = (base: TownBasics | Town) => {
     _economicIdeology: economicIdeology,
     _politicalSource: politicalSource,
     _politicalIdeology: politicalIdeology,
-    _demographicPercentile: {} as Record<RaceName, number>,
-    _baseDemographics: {} as Record<RaceName, number>,
-    get baseDemographics () {
-      console.log('Getting base demographics.')
-      return this._baseDemographics
-    },
-    set baseDemographics (newDemographics) {
-      // lib.setBaseDemographics(this as unknown as Town, newDemographics)
-      console.log('Setting base demographics.')
-      // alert(JSON.stringify(newDemographics))
-      Object.keys(newDemographics).forEach((byRace) => {
-        const race = byRace as RaceName
-        this._baseDemographics[race] = newDemographics[race]
-      })
-    },
-    set demographicPercentile (data) { console.log('Useless demographicPercentile setter. ') },
+    // @TODO: Figure out better way to assign this getter w/o having to have
+    // a setter
+    // eslint-disable-next-line @typescript-eslint/no-empty-function
+    set demographicPercentile (data) {},
```

Comment:
Might be simplest to jsut remove the getter too, and use the lib function directly instead. It's pretty much the same amount of code. Don't need to do that in this PR though

Reply:
Yeah exactly. That will probably automatically fix a whole bunch of issues with saving/loading as getters/setters break serialization.